bridge/src/lib.rs

```diff
@@ -464,16 +469,26 @@ pub mod pallet {
 			// parse proposals and construct signing message
 			let final_message = Self::construct_ecdsa_signing_proposals_data(proposals);
 
-			// recover the signing pubkey
+			// recover the signing address
 			if let Ok(pubkey) =
-				secp256k1_ecdsa_recover_compressed(&sig, &blake2_256(&final_message))
+				// recover the uncompressed pubkey
+				secp256k1_ecdsa_recover(&sig, &blake2_256(&final_message))
```

Comment:
Maybe too cryptologic, why we don't recover a compressed public key, it cost less space.

Reply:
Figure it out too, correct me if I'm wrong: because a compressed public key is produced by the uncompressed public key, so we don't have to do one more step to convert it to a compressed one.